### ISSUE TITLE
Getting -1 for map using VOC07+12 Trainval

### ISSUE THREAD

**Issue by Alina-liu** [xmyqsh/FPN#34]
Hi,thank you for your code,I'm new on this ,I use your net and change the baseline to resnext using resnext50 which is convertied from caffe model ,but I get -1 mAP for all classes,can you tell me why? Thank you and please forgive my poor English!

**Comment by liusai92** [comment]
I encounted this issue too, I checked the weights I trained, there are lots of nan in the scope of res3_5, Top-Down, RPN, Fast-RCNN, is it just a hyperparameter problem?
 I also found that I didn't restore all weights in Resnet50.npy, especially batch-normalization layer's weights 'mean, variance', should them be restored to graph variables beta, gamma?

**Comment by imyhxy** [comment]
The same problem here, which version of tensorflow you are using?

**Comment by rxqy** [comment]
there might some difference between these models
ie, maybe caffe pretrain model requires input normalization. while the origin one doesn't

**Comment by Alina-liu** [comment]
I use tensorflow 1.0.1 ,does this version lead to the problem@imyhxy

**Comment by Alina-liu** [comment]
I used the tensorflow 1.0.1,does this version lead to the problem? @imyhxy

**Comment by imyhxy** [comment]
@Alina-liu I am not sure, but I used the tensorflow 1.8.

**Comment by zhangtongton** [comment]
The same problem here, has it solved? thx @Alina-liu @imyhxy
